src/ansys/mapdl/core/mapdl_grpc.py

```diff
@@ -865,17 +865,12 @@ def _launch(self, start_parm, timeout=10):
 
         # may need to wait for viable connection in open_gui case
         tmax = time.time() + timeout
-        success = False
         while time.time() < tmax:
             try:
                 self.prep7()
-                success = True
                 break
             except:
-                pass
-
-        if not success:
-            raise MapdlConnectionError("Unable to reconnect to MAPDL")
+                raise MapdlConnectionError("Unable to reconnect to MAPDL")
```

Comment:
You are going to raise the exception in the first attempt.... the idea of this while is to wait for a viable connection, and nothing comes up after `timeout`, it will show the exception.

Reply:
Thanks for raising this, I will modify the code accordingly.

Reply:
This has been re-added in https://github.com/ansys/pymapdl/pull/3474/commits/9830881cb0dc880483e33cb41a4da38d4a744b3b. Let me know if the warning message suits you.